src/main/kotlin/app/revanced/patches/youtube/misc/microg/patch/bytecode/MicroGBytecodePatch.kt

```diff
@@ -72,4 +70,15 @@ class MicroGBytecodePatch : BytecodePatch(
 
         return PatchResultSuccess()
     }
+
+    companion object : OptionsContainer() {
+        var packageName: String? by option(
```

Comment:
This option should likely exist for the equivalent patch for YouTube Music. Stronger abstraction is required. Both patches have to be turned into an abstract which then is inherited from both patches. They can then override the execute method each. The abstract patch class can have this companion object.